### PR TITLE
Add digimon showdown chat-plugin

### PR DESCRIPTION
**Pull request by AlphaWind**
Commands to help users to better understand the specifics of each digimon and how they can be used in battle.

### CODE REVIEW

File: wavelength-plugins/digimon.js [hunk 1]
```diff
@@ -0,0 +1,581 @@
...
+	digisearchdisplay: 'digipedia',
+	digisearch: 'digipedia',
+	digimonsearch: 'digipedia',
+	searchchange: 'digipedia',
+	digipedia: function (target, room, user, connection, cmd) {
+		let mod = Dex.mod('digimon');
+		/*
+			/digimonsearch Section:value, Section:value
+			/digisearchdisplay digimon
+			*/
```
Comment (HoeenCoder) [inline wavelength-plugins/digimon.js]: Not sure why travis didn't catch this, should be indented properly.

File: wavelength-plugins/digimon.js [hunk 2]
```diff
@@ -0,0 +1,581 @@
...
+						break;
+					case 'metalgreymonvirus':
+						first = 'MetalGreymon';
+						second = '[Virus]';
+						break;
+					}
+					let type1 = 0;
+					switch (template.types[0]) {
+					case 'Flame':
+						type1 = 'https://i.imgur.com/a6oFkDS.png';
```
Comment (HoeenCoder) [inline wavelength-plugins/digimon.js]: imgur blocked showdown (when trying to load images on showdown web pages, the image will not load with a http 403 error), use the images that zarel uploaded for digimon instead (iirc there are some for the types as well).

File: wavelength-plugins/digimon.js [hunk 3]
```diff
@@ -0,0 +1,581 @@
...
+						type3 = 'https://i.imgur.com/QLFeGOW.png';
+						break;
+					case 'Filth':
+						type3 = 'https://i.imgur.com/X0Fx3FQ.png';
+						break;
+					}
+					let ability = template.abilities[0];
+					switch (ability) {
+					case 'Vaccine':
+						ability = 'https://i.imgur.com/fYpKKQv.png';
```
Comment (HoeenCoder) [inline wavelength-plugins/digimon.js]: Same thing with imgur on abilities too.

File: wavelength-plugins/digimon.js (hunk 2 above)
Comment (Marirapo) [reply wavelength-plugins/digimon.js]: ~~or if needed use discord to host ;X~~

**Pull request merged**